samples/FHIR-Observations-Height-Weight/endpoint.json

```diff
@@ -0,0 +1,160 @@
+{
+  "name": "FHIR - Create Observations (weight/height)",
+  "description": "Endpoint to validate the incoming payload and construct a FHIR Bundle with two Observations for the Height and Weight for the Patient id",
+  "endpoint": {
+    "pattern": "/create-height-weight-observations"
+  },
+  "transformation": {
+    "input": "JSON",
+    "output": "JSON"
+  },
+  "constants": {
+    "resourceType": "Bundle",
+    "type": "transaction",
+    "entry": {
+      "resourceType": "Observation",
+      "status": "final",
+      "request": {
+        "method": "POST",
+        "url": "Observation"
+      },
+      "resource": {
+        "code": {
+          "coding": [
+            {
+              "system" : "http://loinc.org"
+            }
+          ]
+        },
+        "valueQuantity" : {
+          "system" : "http://unitsofmeasure.org"
+        }
+      }
+    },
+    "resourceHeight": {
+      "code": {
+        "coding": [
+          {
+            "code" : "8302-2",
+            "display" : "Body height"
+          }
+        ],
+        "text": "height"
+      },
+      "valueQuantity" : {
+        "unit" : "cm",
+        "code" : "cm"
+      }
+    },
+    "resourceWeight": {
+      "code": {
+        "coding": [
+          {
+            "code" : "29463-7",
+            "display" : "Body Weight"
+          }
+        ],
+        "text": "weight"
+      },
+      "valueQuantity" : {
+        "unit" : "kg",
+        "code" : "kg"
+      }
+    }
+  },
+  "inputValidation": {
+    "type": "object",
+    "properties": {
+      "requestBody": {
+        "type": "object",
+        "properties": {
+          "id": {
+            "type": "string",
+            "pattern": "[A-Za-z0-9\-\.]{1,64}"
```

Comment:
This pattern is missing escapes.

It also fails silently which is an issue. It responds with Bad Request to the user, but there's nothing in the logs at all...

```suggestion
            "pattern": "[A-Za-z0-9\\-\\.]{1,64}"
```

Reply:
:( didnt commit that change

Reply:
I think the Bad Request response is valid because the JSON we are sending isnt valid

Reply:
Cool cool - so it's not even reaching the mediator :+1:

Reply:
Fixed in 9159de3

Reply:
Yeah it doesnt create the endpoint. The error is thrown in the create API when trying to parse the JSON, it throws bad request because the JSON isnt valid